bin/filter.rb

```diff
@@ -24,6 +24,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
 
   doc = Nokogiri::HTML::DocumentFragment.parse(body)
 
+  # Grab footer image for social media previews
+  if !(header =~ /image:/i)
```

Comment:
`header !~ /^image:/i`

The `^` because you only want to match `image:` at the beginning of a line, right?

Reply:
Yep. fixed.